argo-workflows/install-using-helm/ui.md

```diff
@@ -0,0 +1,60 @@
+Helm allows us to configure our deployment by modifying [the default values](https://github.com/argoproj/argo-helm/blob/main/charts/argo-workflows/values.yaml).
+
+The argo-server (and thus the UI) defaults to client authentication, which requires clients to provide their Kubernetes bearer token in order to authenticate. For more information, refer to the [Argo Server Auth Mode documentation](https://argoproj.github.io/argo-workflows/argo-server-auth-mode/). We will switch the authentication mode to server so that we can bypass the UI login for now.
```

Comment:
Can you please bold the first sentence in this paragraph as this one of the key points on this page that users often don't understand/ bother to read the docs to understand? 

Also, can you please add a disclaimer sentence after this paragraph like you've done before for things that are risky for production use? I'm guessing the server auth isn't recommended without more security measures than we're getting into here.